### ISSUE TITLE
Laser reflection

### ISSUE THREAD

**Issue by ppgplayer** [studio-minus/ppg-bugs-and-requests#1331]
**Feature description**
Basically, ALL items reflect lasers. Some reflect them worse, some reflect them better. Like metal is pretty reflectant, but the insulator is less so.

**Feature purpose**
Useful for rangefinders and a bunch of ballistic calculation systems.

**Additional context**
![image](https://user-images.githubusercontent.com/98086053/169646202-5504d1a0-f11a-4a53-b83e-fc7e5d12d8a2.png)

**Comment by DindinYT37** [comment]
I agree but I would suggest to rename this feature request to "Laser reflection rework" or something like that.

**Comment by ppgplayer** [comment]
> I agree but I would suggest to rename this feature request to "Laser reflection rework" or something like that.

I mean, there isn't much to rework since only mirrors reflect stuff currently.

**Comment by DindinYT37** [comment]
> > I agree but I would suggest to rename this feature request to "Laser reflection rework" or something like that.
> 
> I mean, there isn't much to rework since only mirrors reflect stuff currently.

It'd still be a rework of the reflection system, don't you think?

**Comment by ppgplayer** [comment]
> > > I agree but I would suggest to rename this feature request to "Laser reflection rework" or something like that.
> > 
> > 
> > I mean, there isn't much to rework since only mirrors reflect stuff currently.
> 
> It'd still be a rework of the reflection system, don't you think?

no
